exercise_01.py

```diff
@@ -0,0 +1,15 @@
+def displayPattern(n):
+    n_to_print = ""
+    total = ""
+    for i in range(1, n + 1):
+        n_to_print = "{} {}".format(i, n_to_print)
+        space = "  " * (n - i)
+
+        total =  "{}\n{}{}".format(total, space, n_to_print)
+
+    return total
+
+input = eval(input("Enter a number: "))
```

Comment:
Would you mind using `if __name__ == '__main__':`, and would you also mind creating a function called `main` which is called in the previous construct? Please do the same for all other programs.